tests/lora/utils.py

```diff
@@ -633,10 +644,12 @@ def test_simple_inference_with_text_denoiser_lora_save_load(self):
                         check_if_lora_correctly_set(pipe.text_encoder_2), "Lora not correctly set in text encoder 2"
                     )
 
-            images_lora = pipe(**inputs, generator=torch.manual_seed(0)).images
+            images_lora = pipe(**inputs, generator=torch.manual_seed(0))[0]
 
             with tempfile.TemporaryDirectory() as tmpdirname:
-                text_encoder_state_dict = get_peft_model_state_dict(pipe.text_encoder)
+                text_encoder_state_dict = (
```

Comment:
I would only add it to the saving_kwargs if we are testing the text encoder lora, like this
```python
                if self.test_text_encoder_lora:
                    text_encoder_state_dict = get_peft_model_state_dict(pipe.text_encoder)
                    saving_kwargs.update({"text_encoder_lora_layers": text_encoder_state_dict})
```